mcs/tools/macpack/LOADER

```diff
@@ -33,19 +34,24 @@ fi
 
 cd "$APP_PATH/Contents/Resources"
 
+mono4_path=/usr/local/bin/mono
```

Comment:
FYI Mono was always installed in /Library/Frameworks, we just had a symlink from this location before.